docs/changelog.md

```diff
@@ -1,6 +1,40 @@
 Change Log
 ==========
 
+## Version 1.11.0
+
+_2022-03-xx_
+
+Thanks to [@liujingxing][liujingxing] and [@BoD][BoD] for contributing to this release.
+
+* New: Kotlin scripting support in `FileSpec`.
+
+```kotlin
+val spec = FileSpec.scriptBuilder("Taco")
+  .addStatement("println(%S)", "hello world!")
+  .addKotlinDefaultImports()
+  .build()
+```
+
+Generates a `Taco.kts` file with the following contents:
+
+```kotlin
+println("hello!")
+```
+
+* New: Emit trailing commas for multi-line parameters and annotations.
```

Comment:
Unclear if this was new or a fix 😅. CC @JakeWharton

Reply:
New seems fine as it affects the minimum supported Kotlin language version. Not sure if it actually bumps it or not, I suspect this and square brackets for arrays in annotations are competing for the title of defining the minimum.